submissions/the-darkesttt/popup/css/style.css

```diff
@@ -115,28 +134,28 @@
   justify-content: center;
   margin-bottom: 15px;
 }
-.popup__list label[for=inner-popup_checkbox] img,
-.popup .inner-popup_checkbox:checked ~ .inner-popup__list label[for=inner-popup_checkbox] img {
+
+.inner-popup_checkbox_button {
   width: 20px;
 }
-.popup__list label[for=inner-popup_checkbox]:hover,
-.popup .inner-popup_checkbox:checked ~ .inner-popup__list label[for=inner-popup_checkbox]:hover {
-  background-color: #fff;
-  color: var(--show-more);
-}
-.popup__list .inner-popup_checkbox,
-.popup .inner-popup_checkbox:checked ~ .inner-popup__list .inner-popup_checkbox {
-  visibility: hidden;
+
+.inner-popup_checkbox {
```

Comment:
Watch [the video](https://www.youtube.com/watch?v=E6kLaaQFctU&t=413s) to find out how to hide the checkbox in the right way